autotests/qt/meson.build

```diff
@@ -0,0 +1,21 @@
+
+add_languages('cpp', required : true)
+
+qt_dep = dependency('qt5', modules: ['Core', 'Test'])
```

Comment:
What do you think of installing a separate pkg-config file for libportal-qt5?

Reply:
I do install pkg-config file. See libportal-qt/meson.build and in the middle there is 

```
pkgconfig.generate(libportalqt,
  description: 'Qt portal API wrappers',
  name: 'libportal-qt',
  requires: [ qt5_dep ],
)
```

Reply:
I now see what you meant with your question. We now have two pkg-config files since you install one and I install another one for the wrappet. It is slightly confusing and I think I should probably pick a different name for the Qt wrapper. I also think I should rename it to include Qt5 in the name, because one day there might be a Qt6 version.